cmd/kill/kill.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"os"
+	"strconv"
+	"fmt"
+	"flag"
+
+	killer "github.com/tiago4orion/enzo/cmd/kill/killer"
```

Comment:
You can name it `kill` in any package, but if you do so, you can only use inside the same package it was declared (it's not exported).